common/ustring.cpp

```diff
@@ -19,7 +19,7 @@ UString usprintf(const char* fmt, ...)
     UString msg;
     va_list vl;
     va_start(vl, fmt);
-    msg.vsprintf(fmt, vl);
+    msg = msg.vasprintf(fmt, vl);
```

Comment:
Why is this necessary?

Reply:
Newer QT versions like QT 6 don't implement .vsprintf.
This thus needs the newer method.